plugins/webp-uploads/picture-element.php

```diff
@@ -168,3 +168,26 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 		$image
 	);
 }
+
+/**
+ * Adds CSS to make Lightbox trigger BUTTON visible on an Image lightbox when Picture Element is enabled.
+ *
+ * @since n.e.x.t
+ */
+function webp_make_lightbox_trigger_button_visible(): void {
+	if ( has_filter( 'render_block_core/image', 'block_core_image_render_lightbox' ) ) {
```

Comment:
I don't know why, but the `has_filter()` check isn't working. This does though:

```suggestion
	if ( false !== has_action( 'wp_footer', 'block_core_image_print_lightbox_overlay' ) ) {
```